translations/en-us.yml

```diff
@@ -810,6 +810,16 @@ institutions:
             public_projects: 'Public Projects'
             private_projects: 'Private Projects'
             empty: 'No users found matching search criteria.'
+            public_registration_count: 'Public Registrations'
+            private_registration_count: 'Private Registrations'
+            published_preprint_count: 'Preprints'
+            osf_link: 'OSF Link'
+            public_file_count: 'Files on OSF'
+            storage_byte_count: 'Total Data Stored on OSF'
+            account_created: 'Account Created'
+            month_last_login: 'Last Login'
+            month_last_active: 'Last Action'
+            last_action: 'Last Log Action'
```

Comment:
Seems like this isn't used anymore, so we can remove it. Not a blocker for merging though